modules/transports/core/vfs/src/main/java/org/apache/synapse/transport/vfs/PollTableEntry.java

```diff
@@ -515,6 +531,7 @@ public boolean loadConfiguration(ParameterInclude params) throws AxisFault {
     }
 
     protected boolean loadConfigurationsFromService(ParameterInclude params) throws AxisFault {
+        System.out.println("loading config from service");
```

Comment:
Shall we change this to use the logger? Using `System.out.println` to print logs is not encouraged.

Reply:
This lines was added for debugging purpose, we just removed it